ansible_collections/arista/avd/plugins/action/anta_workflow.py

```diff
@@ -183,6 +184,7 @@ def run(self, tmp: Any = None, task_vars: dict | None = None) -> dict:
 
         generate_avd_catalogs = get(PLUGIN_ARGS, "avd_catalogs.enabled")
         structured_config_dir = get(PLUGIN_ARGS, "avd_catalogs.structured_config_dir")
+        user_catalog_enabled = get(PLUGIN_ARGS, "user_catalogs.enabled")
```

Comment:
```suggestion
        generate_user_catalogs = get(PLUGIN_ARGS, "user_catalogs.enabled")
```
The logic below needs to change:

1- If both `generate_avd_catalogs` AND `generate_user_catalogs` are false, we need to raise.
2- We need to raise if `structured_config_dir` and `user_catalog_dir` are None if their respective knob is enabled.

Reply:
Updated the checks. Thanks!!